geograypher/meshes/meshes.py

```diff
@@ -124,6 +124,16 @@ def __init__(
         )
         # Load the texture
         self.logger.info("Loading texture")
+        # load IDs_to_labels
+        # if IDs_to_labels not provided, check the directory of the mesh and get the file if found
+        if IDs_to_labels is None and isinstance(mesh, PATH_TYPE.__args__):
+                possible_json = Path(Path(mesh).stem + '_IDs_to_labels.json')
+                if possible_json.exists():
+                    IDs_to_labels = possible_json
+        # convert IDs_to_labels from file to dict
+        if isinstance(IDs_to_labels, PATH_TYPE.__args__):
+            with open(IDs_to_labels, 'r') as file:
+                IDs_to_labels = json.load(file)
```

Comment:
I think the keys of this dict are going to be `str` typed, not `int`. Maybe there's a way to handle this either when the file is written out or when it's read back in. Otherwise, you'll have to convert it here.